src/api/Gemfile.lock

```diff
@@ -607,4 +609,4 @@ DEPENDENCIES
   yajl-ruby
 
 BUNDLED WITH
-   1.17.2
```

Comment:
Is this change intended?

Reply:
Ops! I just rebased the branch. It's fixed.